Tasks/Common/azure-arm-rest/azure-arm-common.ts

```diff
@@ -221,7 +221,7 @@ export class ApplicationTokenCredentials {
 
         let webRequestOptions: webClient.WebRequestOptions = {
             retriableErrorCodes: null,
-            retriableStatusCodes: [400, 408, 409, 500, 502, 503, 504],
+            retriableStatusCodes: [400, 403, 408, 409, 500, 502, 503, 504],
```

Comment:
Looks like the error is while fetching resource group existence and not while getting access token - and hence making this change would not fix the underlying issue.
Let me look the release failures to see if this is really the case.

Reply:
I double checked and the reason for release failure is that the resource group has been deleted by user (I confirmed in ARM). This matches with my earlier investigation some time back, but in the meanwhile I was blind-sided due to another issue in AKS CD and assumed that this failure is same. The status code did add to this.
We can choose to not make this change. Sorry for the trouble.

Reply:
okay